PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -424,6 +424,25 @@ def miniAOD_customizeCommon(process):
     delattr(process, 'selectedUpdatedPatJetsPuppiJetSpecific')
 
     task.add(process.slimmedJetsPuppi)
+
+    # Embed pixelClusterTagInfos in slimmedJets
+    process.patJets.addTagInfos = True
+    process.patJets.tagInfoSources = cms.VInputTag( cms.InputTag("pixelClusterTagInfos") )
+    process.slimmedJetsNoDeepFlavour.dropTagInfos = '0'
+    process.updatedPatJetsSlimmedDeepFlavour.addTagInfos = True
+    process.updatedPatJetsSlimmedDeepFlavour.tagInfoSources = cms.VInputTag( cms.InputTag("pixelClusterTagInfos") )
```

Comment:
Looking at it more carefully, any changes to `updatedPatJetsSlimmedDeepFlavour` are not needed because TagInfos get added to `updatedPatJetsTransientCorrectedSlimmedDeepFlavour`.

Reply:
Thanks @ferencek , indeed changes to `updatedPatJetsSlimmedDeepFlavour` do not have any effect. They have been removed now also from these lines and the era modifiers, few lines below.